### PR TITLE
Test case for limits page check

### PR DESCRIPTION
**Pull request by fxricky**

### CODE REVIEW

**Comment by bijan2007** [comment]
Please resolve the conflicts and also remove the output files from the commit

**Review by ridhomard — COMMENTED** [review]
Please check indentation

File: src/test/java/testCases/Validate_SelfExlcusion_Limits.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		else{
+			//Increase the current value
+			int temp;
+			temp = Integer.parseInt(Self_Exclusion.txt_MaxAccCashBal(driver).getAttribute("value"));
+			getAccBal = temp + 1;
+			Self_Exclusion.txt_MaxAccCashBal(driver).clear();
+			Self_Exclusion.txt_MaxAccCashBal(driver).sendKeys(Integer.toString(getAccBal));
+			//If the error message is shown
+			if(Self_Exclusion.err_MaxAccBal(driver).isDisplayed()){
+				System.out.println("[P]New amount are not allowed to exceed current amount.");;
```
Comment (ridhomard) [inline src/test/java/testCases/Validate_SelfExlcusion_Limits.java]: Can we put assertEqual() here ? and just getText() for the error message - we can validate the correct error message

File: src/test/java/testCases/Validate_SelfExlcusion_Limits.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	}
+	//Test Method to check whether the number displayed correctly
+	@Test(priority=8)
+	public void Test_Limits_MaxOpenPosition() {
+		Limits.select_MaxOpenPos(driver).getText();
+		Integer.toString(getMaxPos);
+				
+		if(getMaxPos>=60){
+			if(Limits.select_MaxOpenPos(driver).getText().equals(Integer.toString(getAccBal))){
+				System.out.println("Positions in Self-exclusion more than 60, 60 is shown");
```
Comment (ridhomard) [inline src/test/java/testCases/Validate_SelfExlcusion_Limits.java]: This seem doesn't executed ? need to check

File: src/test/java/testCases/Validate_SelfExlcusion_Limits.java [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+	//Test Method to check whether the amount displayed correctly
+	@Test(priority=7)
+	public void Test_Limits_MaxAccountBalance() {
+		Limits.select_MaxAccCashBal(driver).getText();
+		Integer.toString(getAccBal);
+				
+		if(getAccBal>=300000){
+			if(Limits.select_MaxAccCashBal(driver).getText().equals(Integer.toString(getAccBal))){
+				System.out.println("Amount in Self-exclusion more than 300000, 300000 is shown");
+			}
```
Comment (ridhomard) [inline src/test/java/testCases/Validate_SelfExlcusion_Limits.java]: This seem doesn't executed ? need to check

**Comment by fxricky** [comment]
Issues noted. I'm solving the issues.

**Review by bijan2007 — CHANGES_REQUESTED** [review]
Please resolve the conflicts

File: src/test/java/testCases/Validate_Limits.java [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+	   	driver = new ChromeDriver();
+	   	driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+		driver.get(Constant.URL);
+	}
+	//Test Method to close browser session
+	@AfterTest
+	public void endSession() {
+		driver.quit();
+	}
+	// function to convert integer to string with comma
```
Comment (bijan2007) [inline src/test/java/testCases/Validate_Limits.java]: Create a CommonFunctions class under utility package and add this function

**Review by bijan2007 — CHANGES_REQUESTED** [review]
Can you please write all the validation logic in an Action and not inside the test cases ?

**Comment by fxricky** [comment]
Noted, changes in process.

**Review by bijan2007 — APPROVED** [review]

**Pull request merged**